From 7d76799eff16bd893b2c1d9e26b8184caa53ef9a Mon Sep 17 00:00:00 2001
From: Johannes Fischer <johannes.fischer@intel.com>
Date: Wed, 3 Aug 2016 15:40:44 +0200
Subject: [PATCH] fix_missing_concheck

If CONCHECK is not configured also the data related to it is not available. A code bug in NetworkManger causes
a compilation error due to missing declation in this case. The patch takes care of this by just excluding the
respective check.

Signed-off-by: Johannes Fischer <johannes.fischer@intel.com>
---
 src/nm-connectivity.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/nm-connectivity.c b/src/nm-connectivity.c
index b200b51..91eb2e9 100644
--- a/src/nm-connectivity.c
+++ b/src/nm-connectivity.c
@@ -233,9 +233,11 @@ nm_connectivity_check_async (NMConnectivity      *self,
 	g_return_if_fail (NM_IS_CONNECTIVITY (self));
 	priv = NM_CONNECTIVITY_GET_PRIVATE (self);
 
+#if WITH_CONCHECK
 	if (callback == run_check_complete)
 		nm_log_dbg (LOGD_CONCHECK, "Periodic connectivity check started with uri '%s'.", priv->uri);
 	else
+#endif
 		nm_log_dbg (LOGD_CONCHECK, "Connectivity check started with uri '%s'.", priv->uri);
 
 	simple = g_simple_async_result_new (G_OBJECT (self), callback, user_data,
-- 
1.9.1

